templates/components/common/navigation-list-custom.html

```diff
@@ -0,0 +1,111 @@
+            </mask>
+            <g mask="url(#mask0_2973_30873)">
+                <rect x="0.5" width="24" height="24" fill="{{#if is_active}}#5A64C1{{else}}#272727{{/if}}"/>
+            </g>
+        </svg>
+    </a>
+    <div class="c-nav-sub-menu" id="nav-pages-{{id}}" aria-hidden="true" tabindex="-1">
+        <div aria-hidden="true" class="js-quick-search">
+            {{> components/common/quick-search name='nav-menu-quick-search'}}
+        </div>
+        <ul class="c-nav-sub-menu-items">
+            {{#each categories}}
+                {{#each children}}
+                    <li>
+                        {{#if children}}
+                            <a 
+                                class=" js-nav-pages-action has-sub-menu{{#if is_active}} is-active{{/if}} c-nav-page-action c-sub-menu-header-item"
+                                href="{{url}}"
+                                aria-label="{{name}}"
+                            >
+                                {{name}}
+                                <span 
+                                    data-collapsible="nav-pages-{{id}}"
+                                    data-collapsible-disabled-breakpoint="medium"
+                                    data-collapsible-disabled-state="open"
+                                    data-collapsible-enabled-state="closed"
+                                >
+                                    <svg>
+                                        <use href="#icon-chevron-down" />
+                                    </svg>
+                                </span>
+                            </a>
+                            <ul id="nav-pages-{{id}}" class="is-open">
+                                {{#each children}}
+                                    <li>
+                                        <a 
+                                            class="js-nav-pages-action{{#if is_active}} is-active{{/if}} c-nav-page-action c-sub-menu-item"
+                                            href="{{url}}"
+                                            aria-label="{{name}}"
+                                        >
+                                            {{name}}
+                                        </a>
+                                    </li>
+                                {{/each}}
+                            </ul>
+                        {{/if}}
+                    </li>
+                {{/each}}
+            {{/each}}
+            <ul>
+                <li>
+                    <a 
+                        class="js-nav-pages-action c-nav-page-action c-sub-menu-header-item"
+                        href="#"
+                    >
+                        {{lang 'common.login'}}
+                    </a>
+                </li>
+                <li>
+                    <a 
+                        class="js-nav-pages-action c-nav-page-action c-sign-in-now"
```

Comment:
Same here. `c-nav-page-action` and `c-sing-in-now` may overwrite one to another. Also Consider using more descriptive names, and follow BEM structure. For example, it could be c-button but with appropriate modifier class, or if there's more differences create a new component like `c-cta` or `c-nav__cta`if it's more related to the `nav`